File: examples/test_language_filter.php
<?php

/**
 * Process user authentication and validate credentials
 * This function handles user login by checking the provided credentials
 * against the database and returns an authentication token if successful
 */
function authenticate_user(string $username, string $password): string {
    // Implementation details
    return "token123";
}

/**
 * Fetch user profile data from the database
 * Retrieves comprehensive user information including preferences and settings
 * for the specified user identifier
 */
function get_user_profile(int $user_id): array {
    // Implementation details
    return ['id' => $user_id, 'name' => 'User'];
}

/**
 * Calculate order total with tax and shipping
 * Computes the final price including all applicable taxes and shipping costs
 * based on the user's location and selected shipping method
 */
function calculate_order_total(array $items, string $location): float {
    // Implementation details
    return 42.0;
}

/**
 * Send notification email to user
 * Dispatches an email notification to the user's registered email address
 * with the specified subject and message content
 */
function send_email_notification(string $email, string $subject, string $body): bool {
    // Implementation details
    return true;
}

File: examples/test_language_filter.py
"""Test file for language filtering with identical methods across languages"""

def authenticate_user(username: str, password: str) -> str:
    """
    Process user authentication and validate credentials
    This function handles user login by checking the provided credentials
    against the database and returns an authentication token if successful
    """
    # Implementation details
    return "token123"


def get_user_profile(user_id: int) -> dict:
    """
    Fetch user profile data from the database
    Retrieves comprehensive user information including preferences and settings
    for the specified user identifier
    """
    # Implementation details
    return {"id": user_id, "name": "User"}


def calculate_order_total(items: list, location: str) -> float:
    """
    Calculate order total with tax and shipping
    Computes the final price including all applicable taxes and shipping costs
    based on the user's location and selected shipping method
    """
    # Implementation details
    return 42.0


def send_email_notification(email: str, subject: str, body: str) -> bool:
    """
    Send notification email to user
    Dispatches an email notification to the user's registered email address
    with the specified subject and message content
    """
    # Implementation details
    return True